src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -119,6 +134,24 @@ CreateSpaceProcessor::pickHosts(PartitionID partId,
     return pickedHosts;
 }
 
+int32_t CreateSpaceProcessor::GetPartsNumbers() {
```

Comment:
Is this function mean get all part number of all spaces?

Reply:
yeap